// @shopify/cli/dist/node-package-manager-AOVZD6TP.js
import {
  FindUpAndReadPackageJsonNotFoundError,
  PackageJsonNotFoundError,
  UnknownPackageManagerError,
  addNPMDependencies,
  addNPMDependenciesIfNeeded,
  addNPMDependenciesWithoutVersionIfNeeded,
  addResolutionOrOverride,
  bunLockfile,
  checkForNewVersion,
  findUpAndReadPackageJson,
  getDependencies,
  getPackageManager,
  getPackageName,
  getPackageVersion,
  installNPMDependenciesRecursively,
  installNodeModules,
  lockfiles,
  npmLockfile,
  packageManager,
  packageManagerFromUserAgent,
  pnpmLockfile,
  pnpmWorkspaceFile,
  readAndParsePackageJson,
  usesWorkspaces,
  versionSatisfies,
  writePackageJSON,
  yarnLockfile
} from "./chunk-X3OUSYUQ.js";
import "./chunk-PRJ3U7MR.js";
import "./chunk-NLE3ZLU6.js";
import "./chunk-JIJ65JT6.js";
import "./chunk-NO7MYZEO.js";
import "./chunk-GEN4RXTD.js";
import "./chunk-BYC3NA2Y.js";
import "./chunk-SAR5HGRG.js";
import "./chunk-NABCTJNW.js";
import "./chunk-YC4FLMDD.js";
import "./chunk-B7RN7IRD.js";
import "./chunk-LSCXQWNO.js";
import "./chunk-K6257PKA.js";
import "./chunk-XQFDS44U.js";
import "./chunk-AMAYT2P2.js";
import "./chunk-VM4POVNV.js";
import "./chunk-SWNRLCC6.js";
import "./chunk-T5LLJYYS.js";
import "./chunk-ZPL24Y2D.js";
export {
  FindUpAndReadPackageJsonNotFoundError,
  PackageJsonNotFoundError,
  UnknownPackageManagerError,
  addNPMDependencies,
  addNPMDependenciesIfNeeded,
  addNPMDependenciesWithoutVersionIfNeeded,
  addResolutionOrOverride,
  bunLockfile,
  checkForNewVersion,
  findUpAndReadPackageJson,
  getDependencies,
  getPackageManager,
  getPackageName,
  getPackageVersion,
  installNPMDependenciesRecursively,
  installNodeModules,
  lockfiles,
  npmLockfile,
  packageManager,
  packageManagerFromUserAgent,
  pnpmLockfile,
  pnpmWorkspaceFile,
  readAndParsePackageJson,
  usesWorkspaces,
  versionSatisfies,
  writePackageJSON,
  yarnLockfile
};
